bundle.yaml.j2

```diff
@@ -74,6 +74,6 @@ relations:
   - [identity-platform-login-ui-operator:hydra-endpoint-info, hydra:hydra-endpoint-info]
   - [identity-platform-login-ui-operator:ui-endpoint-info, hydra:ui-endpoint-info]
   - [identity-platform-login-ui-operator:ui-endpoint-info, kratos:ui-endpoint-info]
-  - [identity-platform-login-ui-operator:kratos-endpoint-info, kratos:kratos-endpoint-info]
+  - [identity-platform-login-ui-operator:kratos-info, kratos:kratos-info]
```

Comment:
Does this mean we can now remove https://github.com/canonical/kratos-operator/blob/main/charmcraft.yaml#L73?

Reply:
We can remove `kratos-endpoint-info`. Issue opened: https://github.com/canonical/kratos-operator/issues/258